src/Randomizer.App/ViewModels/TrackerMapLocationViewModel.cs

```diff
@@ -51,8 +51,8 @@ public TrackerMapLocationViewModel(TrackerMapLocation mapLocation, TrackerLocati
 
             Locations.ForEach(x =>
             {
-                numClearable += Syncer.IsLocationClearable(x, allowOutOfLogic: false, requireKeys: Region.Name == "Hyrule Castle") ? 1 : 0;
-                numOutOfLogic += Syncer.IsLocationClearable(x, allowOutOfLogic: true) && !Syncer.IsLocationClearable(x, allowOutOfLogic: false) ? 1 : 0;
+                numClearable += Syncer.IsLocationClearable(x, allowOutOfLogic: false, requireKeys: Region.Name == "Hyrule Castle" || syncer.Tracker.World.Config.Keysanity) ? 1 : 0;
```

Comment:
This feels really dirty. One thing for me to keep in mind when I refactor/overhaul things into services: something you can use to access the current config/seed/etc.